ophys_multidepth_mouseids.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "def multidepth_mouseids(visual_area, cre_line):\n",
```

Comment:
you should put this into a .py file rather than a notebook so that others can import it.